packages/models-library/src/models_library/api_schemas_webserver/groups.py

```diff
@@ -75,61 +73,67 @@ class GroupGet(OutputSchema):
 
     @classmethod
     def from_model(cls, group: Group, access_rights: AccessRightsDict) -> Self:
-        # Merges both service models into this schema
+        # Adapts these domain models into this schema
         return cls.model_validate(
             {
-                **_rename_keys(
+                **copy_dict(
```

Comment:
maybe you could have a `remap_keys` argument? but that is ok. Maybe at least add a docstring in the method

Reply:
refactored and moved it into `common_library.dict_tools` together with other similar tools I had around.